README.md

```diff
@@ -8,13 +8,15 @@
 ![min ios version is 13](https://img.shields.io/badge/min%20iOS%20version-13-blue)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-4baaaa.svg)](code_of_conduct.md) 
 
-# Customer.io iOS SDK
+# Summary
```

Comment:
I prefer the previoius "Customer.io iOS SDK" as it is a good header introducing the project. This is a good header for introducing the text below. 

One idea is change the markdown to:

```markdown
# Customer.io iOS SDK

### Summary 
...
```

I like the way we do it in our [ruby SDK](https://github.com/customerio/customerio-ruby). Maybe something similar to that?

Reply:
I totally forgot that the repo title isn't injected at the top of the readme. I like maintaining the semantic order of headings, but this sounds good to me otherwise